api/backup/import.go

```diff
@@ -0,0 +1,167 @@
+package backup
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/boltdb/bolt"
+	"github.com/sirupsen/logrus"
+)
+
+// TODO: use portainer-cli to import
+
+func ImportJsonToDatabase(jsonFilePath, portainerDbPath string) error {
+	if _, err := os.Stat(jsonFilePath); err != nil {
+		return fmt.Errorf("import file not found: %s: %s", jsonFilePath, err)
+	}
+	// if _, err := os.Stat(portainerDbPath); err == nil {
+	// 	return fmt.Errorf("ERROR: database file already exists: %s", portainerDbPath)
+	// }
+
+	return importJson(jsonFilePath, portainerDbPath)
+}
+
+func importJson(jsonFilePath, portainerDbPath string) error {
+	backup := make(map[string]interface{})
+
+	s, err := ioutil.ReadFile(jsonFilePath)
+	if err != nil {
+		return err
+	}
+	//err = json.Unmarshal([]byte(s), &backup)
+	d := json.NewDecoder(bytes.NewReader(s))
+	d.UseNumber()
+	if err = d.Decode(&backup); err != nil {
+		return err
+	}
+
+	connection, err := bolt.Open(portainerDbPath, 0600, &bolt.Options{Timeout: 1 * time.Second})
+	if err != nil {
+		return err
+	}
+	defer connection.Close()
```

Comment:
Is the 1 second timeout enough?   just thinking about network filesystems, especially if we're testing on aws or azure.  In case there is a pause when opening.  They are usually pretty good though.  I would just up it to 2s just to be sure.